avm/res/network/express-route-port/main.bicep

```diff
@@ -0,0 +1,209 @@
+metadata name = 'ExpressRoute Ports'
+metadata description = 'This module deploys an Express Route Port resource used by Express Route Direct.'
+metadata owner = 'Azure/module-maintainers'
```

Comment:
Just a note that the owner metadata has just been removed from all modules (required set-AVMModule readme run)
```suggestion
```

Reply:
removed